services/ui-src/src/components/fields/DynamicField.tsx

```diff
@@ -104,12 +129,21 @@ export const DynamicField = ({ name, label, ...props }: Props) => {
           Add a row
         </Button>
       )}
+      <DeleteDynamicFieldModal
```

Comment:
Nit: The name of this component makes it sound like it deletes the entire DynamicField instead of just one record of the DynamicField. Not sure how I'd change it though, so nbd.

Reply:
Hows DeleteDynamicFieldRecordModal? God its wordy but it tells it like it is

Reply:
lolol that it does. works for me 🤷